crypto/param_build.c

```diff
@@ -159,6 +159,33 @@ int ossl_param_bld_push_BN(OSSL_PARAM_BLD *bld, const char *key,
     return 1;
 }
 
+int ossl_param_bld_push_BN_pad(OSSL_PARAM_BLD *bld, const char *key,
+                               const BIGNUM *bn, size_t sz)
+{
+    int n, secure = 0;
+    OSSL_PARAM_BLD_DEF *pd;
+
+    if (bn != NULL) {
+        n = BN_num_bytes(bn);
+        if (n < 0) {
+            CRYPTOerr(CRYPTO_F_OSSL_PARAM_BLD_PUSH_BN,
+                      CRYPTO_R_ZERO_LENGTH_NUMBER);
+            return 0;
+        } else if (sz < (size_t)n) {
```

Comment:
Why do you have an "else after return" here, but not one at line 179?  Please remove this or (yuk) add it there.

Reply:
Too late in the evening :)